spack/package.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+from importlib.metadata import requires
```

Comment:
The `requires()` used in Spack recipes comes from `spack.package`, so unless I'm missing something I don't think we need this import